lib/WebDriver/AbstractWebDriver.php

```diff
@@ -122,15 +124,30 @@ protected function curl($requestMethod, $command, $parameters = null, $extraOpti
 
         list($rawResult, $info) = ServiceFactory::getInstance()->getService('service.curl')->execute($requestMethod, $url, $parameters, $extraOptions);
 
+        // According to the spec:
```

Comment:
can shorten it.
since this weird behaviour is where it all started, I thought a detailed comment was appropriate